lightning/src/blinded_path/message.rs

```diff
@@ -68,7 +68,6 @@ impl BlindedMessagePath {
 	/// pubkey in `node_pks` will be the destination node.
 	///
 	/// Errors if no hops are provided or if `node_pk`(s) are invalid.
-	//  TODO: make all payloads the same size with padding + add dummy hops
 	pub fn new<ES: Deref, T: secp256k1::Signing + secp256k1::Verification>(
```

Comment:
Call into the new `new_with_dummy_hops` with 0 dummy hops to reduce duplication?